### PR TITLE
hippie meal box revival

### PR DESCRIPTION
**Pull request by SireTurret**
## Changelog
:cl: ArcLumin
add: Added back the Hippie Meal box. There's a toy inside it, too!
/:cl:

## About The Pull Request

rip arclumin
## Why It's Good For The Game

cool item

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
You didn't even try, and no. This was something everyone was saying "no" to even when ol' Arc was still around.

**Comment by karmaisblackandbluepilled** [comment]
what in fuck even is this

**Comment by SireTurret** [comment]
> You didn't even try, and no. This was something everyone was saying "no" to even when ol' Arc was still around.

i did try

**Comment by SireTurret** [comment]
ok no

File: code/datums/components/crafting/recipes.dm [hunk 1]
```diff
@@ -355,6 +355,14 @@
 	parts = list(/obj/item/camera = 1)
 	category = CAT_MISC
 
+/datum/table_recipe/hippiemeal
+	name = "Hippie Meal Box"
+	result = /obj/item/weapon/reagent_containers/food/snacks/hippiemeal
```
Comment (Manly-Dwarf) [inline code/datums/components/crafting/recipes.dm:360]: but like bro if you tried, where's the hippiemeal

Comment (SireTurret) [reply code/datums/components/crafting/recipes.dm:360]: i was about to commit it but im too lazy

**Pull request closed without merging**